tests/inputs/q-search-filter/error_filter_set.py

```diff
@@ -0,0 +1,11 @@
+import django_filters
+from nautobot.apps.filters import NautobotFilterSet
+
+
+class MyAddressObjectFilterSet(NautobotFilterSet):
```

Comment:
```suggestion
class IncorrectQFilterTypeFilterSet(NautobotFilterSet):
```

Reply:
All the existing test classes use the same format of `MyAddress...`